the-blue-alliance-ios/Core Data/EventRanking.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  EventRanking.swift
+//  the-blue-alliance-ios
+//
+//  Created by Bryton Moeller on 6/1/17.
+//  Copyright © 2017 The Blue Alliance. All rights reserved.
+//
+
+import Foundation
+import TBAKit
+import CoreData
+
+extension EventRanking: Managed {
+    
+    var extraStats: [Int] {
+        get {
+            return extraStatsArray as? Array<Int> ?? []
+        }
+        set {
+            extraStatsArray = newValue as NSArray
+        }
+    }
+    
+    var tieBreakers: [Double] {
+        get {
+            return tieBreakersArray as? Array<Double> ?? []
+        }
+        set {
+            tieBreakersArray = newValue as NSArray
+        }
+    }
+    
+    static func insert(with model: TBAEventRanking, for event: Event, for team: Team, in context: NSManagedObjectContext) -> EventRanking {
+        let predicate = NSPredicate(format: "event == %@ AND team == %@", event, team)
+        return findOrCreate(in: context, matching: predicate, configure: { (ranking) in
+            ranking.event = event
+            ranking.team = team
+            
+            if let qualAverage = model.qualAverage {
+                ranking.qualAverage = NSNumber(value: qualAverage)
+            }
+            
+            ranking.rank = Int16(model.rank)
+            
+            if let wins = model.record?.wins {
+                ranking.wins = wins as NSNumber
```

Comment:
For all of these (`wins`, `ties`, `losses`, `dq`, `matchesPlayed`) - can we create an NSNumber object instead of casting